src/renderer/bootstrap.tsx

```diff
@@ -1,21 +1,22 @@
-import "./components/app.scss";
-
-import React from "react";
 import * as Mobx from "mobx";
 import * as MobxReact from "mobx-react";
-import * as LensExtensions from "../extensions/extension-api";
-import { App } from "./components/app";
-import { LensApp } from "./lens-app";
+import React from "react";
 import { render, unmountComponentAtNode } from "react-dom";
-import { isMac } from "../common/vars";
+import { clusterStore } from "../common/cluster-store";
 import { userStore } from "../common/user-store";
+import { isMac } from "../common/vars";
 import { workspaceStore } from "../common/workspace-store";
-import { clusterStore } from "../common/cluster-store";
-import { i18nStore } from "./i18n";
-import { themeStore } from "./theme.store";
-import { extensionsStore } from "../extensions/extensions-store";
+import * as LensExtensions from "../extensions/extension-api";
+import { extensionDiscovery } from "../extensions/extension-discovery";
 import { extensionLoader } from "../extensions/extension-loader";
+import { extensionsStore } from "../extensions/extensions-store";
 import { filesystemProvisionerStore } from "../main/extension-filesystem";
+import { App } from "./components/app";
+import "./components/app.scss";
+import { i18nStore } from "./i18n";
+import { LensApp } from "./lens-app";
+import { themeStore } from "./theme.store";
```

Comment:
Did you sort these? Because I think that one big block of imports doesn't make it easier to read. If we do sort them then I would prefer to keep the `scss` file at the top (since that is the style we seem to have in this repo, which is nice imo). and then have all the external deps in one block and then have all the other deps in another block.

Reply:
I always use TypeScript feature "Organize imports". We could enable eslint rule for consistency though.

Reply:
I think an eslint rule would be good, but in a different PR.

Reply:
I'm also upvoting for using organizing imports utilities but keeping scss and `React` imports at the top.

Reply:
Moved scss / React to the top.